source/application.py

```diff
@@ -1,13 +1,49 @@
+
+import unittest
+
+import sys
+from coverage import coverage
+
 from flask import Flask
+from flask_script import Manager
 from werkzeug.contrib.fixers import ProxyFix
-
 from endpoints.api import api
 
 app = Flask(__name__)
 app.wsgi_app = ProxyFix(app.wsgi_app)
 
 api.init_app(app)
 
-if __name__ == "__main__":
-    """ Setting host to 0.0.0.0 makes it available within the network. """
+manager = Manager(app)
+
+@manager.command
+def run():
+    app.run(host="0.0.0.0", port=8000)
```

Comment:
Have you looked into [gunicorn](http://gunicorn.org/)? It will allow you to handle multiple users better than how flask does it: one request at a time. In other words it allows for concurrent users.